reinforcement-learning/_posts/2018-12-17-tracto_td_learning.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: review
+title: "Tractography using reinforcement learning and adaptive-expanding graphs"
+tags: reinforcement-learning tractography
```

Comment:
The *reinforcement-learning* tag is not currently listed; we can either include it, modify the existing *reinforcement*, or else, use the latter in the review.

Reply:
Changed it for _reinforcement_